ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md

```diff
@@ -905,6 +905,12 @@ roles/eos_cli_config_gen/docs/tables/router-ospf.md
 roles/eos_cli_config_gen/docs/tables/router-path-selection.md
 --8<--
 
+### Router segment-security
```

Comment:
This is under the level-2 header of "Routing". I think we should find a better "home" for it.

Reply:
see line 956 `## Security`

Reply:
Agreed - will fix.